graf3d/eve7/src/REveCaloData.cxx

```diff
@@ -818,6 +808,10 @@ void REveCaloDataHist::GetCellData(const REveCaloData::CellId_t &id,
 
 Int_t REveCaloDataHist::AddHistogram(TH2F* hist)
 {
+    if (!fEtaAxis) {
+      fEtaAxis.reset(hist->GetXaxis());
+      fPhiAxis.reset(hist->GetYaxis());
+   }
```

Comment:
This is wrong. Assign pointer to unique_ptr one also delegates ownership.
Here one have to create copy of TAxis

Reply:
@osschar, same problem is in EVE,